orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/QualityControlChapter.java

```diff
@@ -256,4 +259,31 @@ private void addSageBQRPlots(@NotNull Document document)
         tumorImage.setHorizontalAlignment(HorizontalAlignment.CENTER);
         document.add(tumorImage);
     }
+
+    private void addTumorStats(@NotNull Document document)
+    {
+        Cells cells = new Cells(reportResources);
+        Table tumorStats = new Table(UnitValue.createPercentArray(new float[] { 3, 1 })).setWidth(contentWidth());
+
+        tumorStats.addCell(cells.createContent("Tumor maximum diploid proportion"));
+        tumorStats.addCell(cells.createContent(formatTwoDigitDecimal(report.purple().tumorStats().maxDiploidProportion())));
+
+        tumorStats.addCell(cells.createContent("Number of hotspot mutations"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().hotspotMutationCount())));
+
+        tumorStats.addCell(cells.createContent("Number of hotspot structural variants"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().hotspotStructuralVariantCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of small variants allele read counts"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().smallVariantAlleleReadCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of structural variant tumor fragment counts (excluding single breakends)"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().structuralVariantTumorFragmentCount())));
+
+        tumorStats.addCell(cells.createContent("Sum of B-allele frequency points in germline diploid regions regions with tumor ratio < 0.8 OR > 1.2"));
+        tumorStats.addCell(cells.createContent(String.valueOf(report.purple().tumorStats().bafCount())));
+
+        document.add(new Tables(reportResources).createWrapping(tumorStats,
+                "Tumor detection requirements", "At least one requirement needs to be met to indicate detection of tumor, in case of highly diploid candidate solutions (proportion >=0.97)\n\n"));
+    }
```

Comment:
We might want to be careful with this subtitle for two reasons:
- In targeted and/or tumor-only mode the FAIL_NO_TUMOR check is completely different.
- Even when passing these requirements, a FAIL_NO_TUMOR can still be called if the somatic purity fit fails.

Reply:
So for the reporting section the remaining loose ends appear to be the wording of this, and if we should again include the count thresholds as mentioned earlier or if its ok to just display the observed counts. Welcome recommendations on this :smile:

Reply:
see mail!